File: week01/004.forloop.rs
// Summation with `for`
#![feature(step_by)]

use std::io;


fn main() {
    let mut sum: u32 = 0;

    for i in 2..102 {
        if i%2 == 0 {
            sum +=i;
        }
    }

    println!("stable version: Sum is {}  ", sum);

    sum =0;
    for i in (2..102).step_by(2) {
            sum +=i;
    }

    println!("nightly version: Sum is {}  ", sum);
}

File: week01/004.forloop.c
// Summation with `for`
#include <stdio.h>

int main(void) {
    unsigned int sum = 0;
    unsigned int i;

    for(i=2; i <= 100; i +=2) {
        sum +=i;
    }

    printf("Sum is %u\n", sum);
}
